### PR TITLE
Adds Griddle and Usable Cleanbot to Space Bar

### PR DESCRIPTION
**Pull request by Aquizit**
As the title says.
Griddle wouldn't fit in current kitchen size, expanded the Space Bar kitchen by 1 tile on the y-axis to make it all fit.

Also added a Space Bar cleanbot and put it in the bar as the space bartender couldn't use his, and adding his access to the normal bot would let station bartenders control all the station ones.

:cl:  
rscadd: Adds griddle to Space Bar
rscadd: Adds new cleanbot for Space Bar that the bartender can use
tweak: Expanded Space Bar kitchen very slightly to make it fit
/:cl:

### CODE REVIEW

**Review by Aquizit — COMMENTED** [review]
Fixing all the drone shit I was testing. I forgot I was in this branch.

**Review by patpol4 — APPROVED** [review]
Map is fine, still need a maintainer to check the code part

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm [hunk 1]
```diff
@@ -316,3 +316,11 @@ Maintenance panel panel is [open ? "opened" : "closed"]"})
...
     . = ..()
     bot_core.req_one_access = list(ACCESS_JANITOR, ACCESS_ROBOTICS, ACCESS_MEDICAL)
+
+/mob/living/simple_animal/bot/cleanbot/spacebar
+    name = "Frank Cleansington III"
+    desc = "A little cleaning robot, he looks so excited! You still have no idea why your dad named it this."
+
+/mob/living/simple_animal/bot/cleanbot/spacebar/Initialize()
+	. = ..()
+	bot_core.req_one_access = list(ACCESS_BAR)
```
Comment (adamsong) [inline code/modules/mob/living/simple_animal/bot/cleanbot.dm]:
```suggestion
	bot_core.req_access = list(ACCESS_BAR)
```

**Review by adamsong — DISMISSED** [review]

File: config/admins.txt [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 AshCorr	=	Host
 Xantam	=	Host
- 
+
```
Comment (adamsong) [inline config/admins.txt]:
```suggestion
 
```

**Pull request merged**